### DISCUSSION TITLE
Mined block rewards decline

### DISCUSSION THREAD

**Discussion by hajes** [Chia-Network/chia-blockchain#14385] [General]
Just three months ago, I used to hit on average 4 plots per month. Last three months, average is 3 blocks per 3 months.

Are you going Ethereum way of killing miners/farmers who invested in your network >30k $?

**Comment by juelianfy** [comment]
The decrease in income is because many people have cultivated the compressed land, and our income will drop significantly if we do not cultivate the compressed land. The official has not released hoes that support the compressed land.

**Reply by hajes** [reply]
Why do you allow unofficial crap in official network?

It is like cheat codes in games ;-) Who will use a network where every user can change whatever she wants?

NetSpace is still in decline. I still have the same estimate for winning blocks. I doubt it is a compression problem. It is not an isolated incident either - there are other similar size farmers, experiencing the same problem.
